velox/connectors/tpch/TpchConnector.cpp

```diff
@@ -127,21 +128,31 @@ void TpchDataSource::addSplit(std::shared_ptr<ConnectorSplit> split) {
       "Previous split has not been processed yet. Call next() to process the split.");
   currentSplit_ = std::dynamic_pointer_cast<TpchConnectorSplit>(split);
   VELOX_CHECK(currentSplit_, "Wrong type of split for TpchDataSource.");
-  splitOffset_ = 0;
+
+  size_t partSize =
```

Comment:
Can we rename `partSize` to something else? The `part` here can be confused with the TPCH table `parts`. At the first glance I was thinking this is related to the size of the parts table. Same for other places. How about `batch` or some other names?

Reply:
I'm trying to follow the same semantic as the tpch splits in Presto, as that's that they call it there =/